packages/ethereum-contracts/contracts/mocks/SuperAppBaseFlowMock.sol

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: AGPLv3
+pragma solidity >= 0.8.0;
+
+import {ISuperfluid, ISuperToken, ISuperApp, SuperAppDefinitions} from "../interfaces/superfluid/ISuperfluid.sol";
+import {SuperTokenV1Library} from "../apps/SuperTokenV1Library.sol";
+
+abstract contract SuperAppBaseFlowMock is ISuperApp {
```

Comment:
this contract should inherit from SuperAppBaseFlow instead of duplicating its code, otherwise the tests aren't testing the actual code, but a copy of it (wihch may get out of sync).